db/data_migration/20150219120320_add_historic_governments.rb

```diff
@@ -1,5 +1,5 @@
 governments = [
-  ["2010 to 2015 Conservative and Liberal democrat coalition government", "2010-05-12"],
+  ["2010 to 2015 Conservative and Liberal Democrat Coalition government","2010-05-12"],
```

Comment:
I see why we’ve capitalised “Liberal Democrat” (they are a proper noun, after all), but why capitalise “Coalition”? Especially since we aren’t capitalising the coalition between 1918 and 1922.